### ISSUE TITLE
How to set learning_rate when using pretrained weights

### ISSUE THREAD

**Issue by Holidays1999** [microsoft/Swin-Transformer#126]

**Comment by ancientmooner** [comment]
Try different lr and evaluate on your datasets. For classification, you can start by trying 1/10 of the pre-training learning rate
